astropy/modeling/core.py

```diff
@@ -3384,14 +3483,39 @@ def make_subtree_dict(tree, nodepath, tdict, leaflist):
         tdict[nodepath] = (tree, leftmostind, rightmostind)
 
 
-_ORDER_OF_OPERATORS = [('|',), ('&',), ('+', '-'), ('*', '/'), ('**',)]
+_ORDER_OF_OPERATORS = [('fix_inputs',),('|',), ('&',), ('+', '-'), ('*', '/'), ('**',)]
 OPERATOR_PRECEDENCE = {}
 for idx, ops in enumerate(_ORDER_OF_OPERATORS):
     for op in ops:
         OPERATOR_PRECEDENCE[op] = idx
 del idx, op, ops
 
 
+def fix_inputs(modelinstance, values):
+    """
+    This function creates a compound model with one or more of the input
+    values of the input model assigned fixed values (scalar or array).
+
+    Parameters
+    ----------
+    modelinstance: Model instance. This is the model that one or more of the
+        model input values will be fixed to some constant value.
+    values: A dictionary where the key identifies which input to fix
+        and its value is the value to fix it at. The key may either be the
+        name of the input or a number reflecting its order in the inputs
+
+    Examples
+    --------
+
+        >>> from astropy.modeling.models import Gaussian2D
+        >>> g = Gaussian2D(1,2,3,4,5)
+        >>> gv = fix_inputs(g, {0: 2.5})
```

Comment:
The documentation build is failing probably because of the wrong indentation in the `Examples` section (should be aligned with the section title).